projects/ngrx-auto-entity/src/lib/factories/util.ts

```diff
@@ -0,0 +1,32 @@
+import { ActionCreator, Creator } from '@ngrx/store';
```

Comment:
This file is where the real "magic" occurs. This file in particular was CRITICAL to attaining compatibility with NgRx 8+ and its `ActionCreator` and `Creator` types, which are foundational to all of the strong action/action factory typing involved in `ofType`, `on` of the `createReducer` function, etc.